cli/__tests__/logger.test.js

```diff
@@ -9,21 +9,28 @@ describe('logger', () => {
   const oneArgMethods = [
     { method: 'log', expectedPrefix: '' },
     { method: 'task', expectedPrefix: '👍  ' },
-    { method: 'start', expectedPrefix: '🔥  ' },
-    { method: 'end', expectedPrefix: '✅  ' },
+    { method: 'start', expectedPrefix: '\n🔥  ' },
+    { method: 'end', expectedPrefix: '\n✅  ' },
     { method: 'info', expectedPrefix: 'ℹ️  ' },
   ];
 
   const twoArgMethods = [
     { method: 'warn', expectedPrefix: '🙀  ' },
-    { method: 'error', expectedPrefix: '❌  ' },
+    { method: 'error', expectedPrefix: '\n❌  ' },
     { method: 'debug', expectedPrefix: '🐞  ' },
   ];
 
+  const newlineMethods = ['start', 'end', 'error'];
+
   [...oneArgMethods, ...twoArgMethods].forEach(({ method, expectedPrefix }) => {
     it(`logger.${method}: simple usage`, () => {
       logger[method]('here is some text');
-      expect(global.console.log.mock.calls).toEqual([[`${expectedPrefix}here is some text`]]);
+      if (newlineMethods.indexOf(method) > -1) {
+        expect(global.console.log.mock.calls)
+          .toEqual([[`${expectedPrefix}here is some text`], []]);
+      } else {
+        expect(global.console.log.mock.calls).toEqual([[`${expectedPrefix}here is some text`]]);
```

Comment:
@tizmagik the difference here is that this also verifies it was called exactly once, with only that one argument... could add some more assertions to get that too, maybe this is too opaque